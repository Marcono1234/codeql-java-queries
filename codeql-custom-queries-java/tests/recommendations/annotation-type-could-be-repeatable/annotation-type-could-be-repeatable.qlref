./recommendations/annotation-type-could-be-repeatable.ql
