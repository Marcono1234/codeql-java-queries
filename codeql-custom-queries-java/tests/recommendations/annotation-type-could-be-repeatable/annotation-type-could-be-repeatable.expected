| Test.java:6:16:6:19 | Anns | Could be used as containing annotation type by marking $@ as @Repeatable | Test.java:4:16:4:18 | Ann | @Ann |
